aws_lambda_powertools/utilities/typing/event.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+from typing import Any, Dict
+
+LambdaEvent = Dict[str, Any]
```

Comment:
@Nr18 - this one come with minimal benefit. I am sure people will just include LambdaContext and leave event as a `Dict`, it is easier to add at a glance what it contains.

```python3
from aws_lambda_powertools.utilities.typing import LambdaContext


def handler(event Dict[str, Any], context LambdaContext):
    pass
```

Reply:
Yeah, I was thinking of that indeed especially because I had a issue with the import order when the file called `lambda_event.py`.

Reply:
@michaelbrewer I refactored it in 872f99c

Reply:
You may want to remove references to `LamdbaEvent` also from the `utilities_typing.png` file :)

Reply:
🤦 thanks!